lib/wanda/executions/server.ex

```diff
@@ -36,6 +36,9 @@ defmodule Wanda.Executions.Server do
   """
   @impl true
   def start_execution(execution_id, group_id, targets, env, config \\ []) do
+    # TODO: this should be removed when web properly sets the target type in ExecutionRequested event
```

Comment:
Personally I wouldn't include this little change here, as it is a bit hidden.
But it is really small, and I understand the incovenience of creating a PR for only this.
Anyway, are we in the need to add this "hidden" step instead of having the actual implementation?

Reply:
Thing is that if we don't provide the target type when building the ExecutionCompleted event, it would break.

The actual implementation where web sets target type in ExecutionRequested event is part of another story (actually starting a host check execution), which we're gonna address in next iteration.

Just a matter of scope. Here we we're focusing on allowing the selection of host checks. Next we'll introduce execution of host checks.

Would this work?

Reply:
Alright, so "yes", we need it to move the PR forward hehe
Get it!
Yes, go on. I guess it will be changed really soon